torch/_inductor/ir.py

```diff
@@ -2052,13 +2109,16 @@ def as_storage_and_layout(
         if freeze:
             if want_contiguous:
                 x.data.freeze_layout()
-                assert x.data.layout.is_contiguous()
+                assert (
```

Comment:
if `x` is an IRNode and `get_layout` was typed as returning a Layout, you would not need this isisntance